services/order.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+from typing import List
+from django.db.models import QuerySet
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(tickets: List[dict], username: str, date: str = None) -> None:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]